src/components/Banner/Banner.tsx

```diff
@@ -46,7 +46,7 @@ export interface Props {
    * - **error** - results in a red banner
    * - **info** - results in a blue banner
    */
-  variant?: 'success' | 'warning' | 'error' | 'info' | 'brand';
+  variant?: 'success' | 'warning' | 'error' | 'info' | 'brand' | 'vertical';
```

Comment:
I don't think we want `vertical` to be a type of `variant` because a banner can be vertical and also have one of the other variants (success, warning, etc).

Reply:
@dierat Yep I agree. Hey, this is a good exercise! May I recommend capturing these as tickets or something so we can address that cleanup independently of this PR? This PR is just flipping things around, but I think it makes sense to use this as an audit exercise and tackle any issues outside of this PR.

Reply:
I'd generally agree with making tickets for pre-existing issues. I think in this case though there was a typo below:
```
[styles['banner--vertical']]: orientation === 'vertical',
```
turned into
```
variant === 'vertical' && styles['banner--vertical'],
```
, where `variant` should have been `orientation` if things were properly flipped. I bet the props change was to resolve the resulting TS issue (and it's probably what happened in the `align` case as well)

Reply:
@anniehu4 Yes, this was the result of a typo. I inadvertently changed:

`[styles['banner--vertical']]: orientation === 'vertical'` 

into

`variant === 'vertical' && styles['banner--vertical']`

which then generated a TypeScript error:

`This condition will always return 'false' since the types '"success" | "warning" | "error" | "info" | "brand"' and '"vertical"' have no overlap.`

Rather than realize that I had a typo in the refactored logic, I mistakenly corrected the TS error by piping "vertical" into the type union assigned to `variant`. My bad.